src/mutations/item.ts

```diff
@@ -433,19 +438,22 @@ export default (queryClient: QueryClient, queryConfig: QueryClientConfig) => {
             item.set(
               'path',
               buildPath({
-                prefix: queryClient.getQueryData<Record<Item>>(buildItemKey(to))?.get('path') ?? '',
+                prefix:
+                  queryClient
```

Comment:
Can we use the default for `get()`, i.e. `get('key', defaultValue)`?

Reply:
Instead of `??`

Reply:
Actually  this trick (default value using `get`) works only if the key does not exist, so it's still possible to have an `undefined` value. Related to that, typescript requires a default value, so I would anyway need `?? ''`